spec/models/question_spec.rb

```diff
@@ -3,7 +3,11 @@
 require 'rails_helper'
 
 RSpec.describe Question, type: :model do
-  it { is_expected.to validate_presence_of :intitule }
+  it { should validate_presence_of :intitule }
```

Comment:
pourquoi avoir changé ça ?
c'est la nouvelle manière de faire, le `should` est deprecated je crois

Reply:
C'est mélangé dans tout le code, parfois on utilise `is_expected.to`, parfois `should`

Reply:
Je vois pas de règle Rubocop qui indique qu'il faut utiliser l'un plutôt que l'autre

Reply:
En regardant la gem `shoulda-matcher`, il parle de ce problème : 

https://github.com/thoughtbot/shoulda-matchers#should-vs-is_expectedto

Donc : 
1. soit on préfère garder le should car plus court
2. soit on utilise is_expected et on supprime la gem `shoulda-matcher`

Reply:
je vote pour le 2.

Reply:
J'ai donc fait le choix 2 et est harmonisé la manière de faire dans l'appli